src/openzaak/components/besluiten/tests/test_besluit_create.py

```diff
@@ -114,6 +114,7 @@ def test_us162_voeg_besluit_toe_aan_zaak(self):
                 besluit.besluitinformatieobject_set.get().informatieobject, io.canonical
             )
 
+    @override_settings(ALLOWED_HOSTS=["openzaak.nl", "testserver"])
```

Comment:
not sure what exactly makes these changes required?

Reply:
for some reason I was getting 400's due to disallowed host when testing locally, I'm not sure why that was happening though

Reply:
Can you try with a local `.env` file?

```bash
(open-zaak) ➜  open-zaak git:(master) cat .env
DEBUG=1
ALLOWED_HOSTS=localhost,127.0.0.1,testserver,openzaak.nl,example.com
DB_NAME=utrecht
```

Reply:
thanks, that fixes it